test/lib/builder/builder.js

```diff
@@ -342,7 +342,8 @@ test("Build theme.j even without an library", (t) => {
 	const expectedPath = "./test/expected/build/theme.j/dest";
 	return builder.build({
 		tree: themeJTree,
-		destPath
+		destPath,
+		excludedTasks: ["escapeNonAsciiCharacters"]
```

Comment:
Why is the task excluded here? I would suspect the theme to not have any .properties files anyways

Reply:
done

Reply:
fixed with: 30ada8b